plugins/pulp_ostree/plugins/importers/steps.py

```diff
@@ -182,20 +195,23 @@ def process_main(self, item=None):
         """
         lib_repository = lib.Repository(self.parent.storage_dir)
         for ref in lib_repository.list_refs():
+            # the branches listed here can have an undesired prefix ending with a ":"
+            branch = ref.path.split(':')[-1]
```

Comment:
refs here were showing up with a prefix of ``fedora:`` that was not present in the user input examples. A user would specify branch ``fedora-atomic/f23/x86_64/docker-host``, which would not be sync'd because it was listed here as ``fedora:fedora-atomic/f23/x86_64/docker-host``. I'm hoping @jortel can weigh in on whether this is the right way to fix that problem, even if he does so in January after this gets merged.